understanding/22/pointer-target-spacing.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+   <meta charset="UTF-8"></meta>
+   <title>Understanding Pointer Target Spacing</title>
+   <link rel="stylesheet" type="text/css" href="../../css/editors.css" class="remove"></link>
+</head>
+<body>
+   <h1>Understanding Pointer Target Spacing</h1>
+   
+    <section id="status" class="advisement">
+        <h2>Status</h2>
+        <p>This understanding document is part of the <a href="https://w3c.github.io/wcag/guidelines/22/"><strong>draft</strong> WCAG 2.2 content</a>. It may change or be removed before the final WCAG 2.2 is published.</p>
+    </section> 
+
+    <section class="remove">
+   
+        <h2>Pointer Target Spacing Success Criteria text</h2>
+        <blockquote>
+            <h4>Pointer Target Spacing</h4>
+    
+            <p class="conformance-level">AA</p>
+            <p class="change">New</p>
+            
+            <p>For each target, there is an area with a width and height of at least 44 CSS pixels that includes it, and no other targets, except when:</p>
+            <ul>
+                <li><strong>Enlarge:</strong> A mechanism is available to change the CSS pixel size of all targets so that the width and height are each at least 44 CSS pixels;</li>
+                <li><strong>Inline:</strong> The target is in a sentence or block of text;</li>
+                <li><strong>Essential:</strong> A particular presentation of the target is essential to the information being conveyed.</li>
+            </ul>
+            <p class="note">This criteria has been formulated to increase the hit-area of small targets, but the group would like feedback from providers of touch-screen devices if there is another way of forming the criteria to better complement the tap-heuristics used.</p>
+            <p class="note">Are there issues with internationalization when describing inline links?</p>
+            <p class="note">Are there issues with pop-over content overlapping targets triggering failures?</p>
+        </blockquote>
+    </section>
+   
+    <section id="intent">
+        <h2>Intent of Pointer Target Spacing</h2>
+
+        <p>The intent of this success criteria is to ensure targets can easily be activated without accidentally activating an adjacent target. When touch targets are small, it is difficult for users with hand tremors and those who have difficulty with fine motor movement to activate them accurately. Providing spacing between touch targets will reduce the likelihood of accidentally activating the wrong control.</p>
```

Comment:
"this success criteria" > "this success criterion" (singular)

change mention of "touch targets". this is about targets in general (touch, mouse, stylus, etc).